### DISCUSSION TITLE
Chart was unable to show up correctly while using WPF sample.

### DISCUSSION THREAD

**Discussion by roface** [Live-Charts/LiveCharts2#1567] [Q&A]
I followed the website and made a complete same WPF sample.
[https://livecharts.dev/docs/WPF/2.0.0-rc2/Overview.Installation](url)
But somehow the chart didn't show up when I start the application.

[ViewModel.cs]

````
using LiveChartsCore;
using LiveChartsCore.SkiaSharpView;

namespace WpfSample
{
    public class ViewModel
    {
        public ISeries[] Series { get; set; }
            = new ISeries[]
            {
                new LineSeries<double>
                {
                    Values = new double[] { 2, 1, 3, 5, 3, 4, 6},
                    Fill = null
                }
            };
    }
}
````

[MainWindow.xaml]
````
<Window x:Class="WpfSample.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:local="clr-namespace:WpfSample"
        xmlns:lvc="clr-namespace:LiveChartsCore.SkiaSharpView.WPF;assembly=LiveChartsCore.SkiaSharpView.WPF"
        mc:Ignorable="d"
        Title="MainWindow" Height="450" Width="800">

    <Window.DataContext>
        <local:ViewModel />
    </Window.DataContext>

    <lvc:CartesianChart
        Series="{Binding Series}">
    </lvc:CartesianChart>
</Window>
````

First of all, build and start the applcation but got a whole blank screen.
![lv1](https://github.com/user-attachments/assets/c2c2ed96-a23d-4bd8-9cc5-0ecc502006bf)

When I click "maximize" button on window, the chart showed up.
![lv2](https://github.com/user-attachments/assets/d086b12a-a70d-49dd-a571-949d8cad98dd)

Then click "Restore" button on window, the window returned to normal size and the chart left axis line only.
![lv3](https://github.com/user-attachments/assets/38b32b3f-46b7-4a14-ab56-b95fa88fe2e8)

A complete project files has been zip and uploaded to google drive.
[https://drive.google.com/file/d/1KlJRkpG38tBk3S_pxETvnxcdO_9VrSoB/view?usp=sharing](url)

Could someone help me to apply LiveCharts2 on WPF ?

**Comment by Daimonion1980** [comment]
Your sample works flawlessly on my system. 

Thought it may be a bug similar to my [posted one](https://github.com/beto-rodriguez/LiveCharts2/issues/1505) but every resizing is perfect in your sample.

Are you able to test it on another installation?

**Comment by roface** [comment]
It's Graphic driver issue.
[](https://github.com/beto-rodriguez/LiveCharts2/issues/1566)

**Reply by roface** [reply]
Please check the link below for more details.
https://github.com/beto-rodriguez/LiveCharts2/issues/1566
